pkg/expect/expect.go

```diff
@@ -122,22 +124,82 @@ func (ep *ExpectProcess) read() {
 func (ep *ExpectProcess) tryReadNextLine(r *bufio.Reader) error {
 	printDebugLines := os.Getenv("EXPECT_DEBUG") != ""
 	l, err := r.ReadString('\n')
-
+	f, err1 := os.Create("logs.txt")
```

Comment:
What happens if file exits? Can you rewrite it so it:
* Doesn't write to current directory but to test tmp directory like `t.TempDir`.
* Each etcd cluster member writes to separate file that can be recognised by name e.g. `member-1.log`
*  As member process can be restarted, we should not override the file but append to it's end.